resources/dicts/patrols/new_cat_hostile.json

```diff
@@ -51,93 +96,161 @@
     "chance_of_success": 40,
     "exp": 10,
     "win_skills": ["smart", "very smart", "extremely smart"],
-    "min_cats": 1,
+    "min_cats": 2,
     "max_cats": 6
 
+},
+{
+    "patrol_id": "gen_bord_newcatthunderpathsolo2",
+    "biome": "Any",
+    "season": "Any",
+    "tags": ["border", "new_cat", "new_cat_injury", "nc_blunt_force_injury", "reputation", "no_change_fail_rep"],
+    "intro_text": "p_l finds a wounded cat near the Thunderpath.",
+    "decline_text": "p_l leaves the wounded cat alone.",
+    "success_text": [
+        "p_l approaches the cat and sees some signs of life. After some hesitation, {PRONOUN/p_l/subject} {VERB/p_l/decide/decides} to bring the cat back to camp so that they can be treated by the medicine cat. The cat thanks {PRONOUN/p_l/object} profusely and promises to repay {PRONOUN/p_l/object} by joining the Clan and being the best warrior they can."
+    ],
+    "fail_text": [
+        "As p_l inspects the cat, {PRONOUN/p_l/subject} find that they are already hunting with their ancestors.",
+        "p_l attempts to carry the cat back to camp, but the cat passes on the way. {PRONOUN/p_l/subject/CAP} {VERB/p_l/stop/stops} to bury the cat before continuing home."
+    ],
+    "antagonize_text": "p_l drives the cat off of the territory. It's not c_n's problem.",
+    "antagonize_fail_text": "p_l tries to drive the cat away, but they trip and stumble over a loose twig. They lay on the ground and don't get back up. p_l takes a bit of time to bury them - just to keep the scavengers away.",
+    "chance_of_success": 40,
+    "exp": 10,
+    "win_skills": ["smart", "very smart", "extremely smart"],
+    "min_cats": 1,
+    "max_cats": 1
+
 },
 {
     "patrol_id": "gen_bord_newkit3",
     "biome": "Any",
     "season": "Any",
     "tags": ["border", "new_cat_kitten", "reputation", "no_change_fail_rep"],
     "intro_text": "r_c finds an abandoned kit whose parents are nowhere to be found.",
+    "decline_text": "They decide to leave the kit alone.",
     "success_text": [
         "The kit is taken back to camp and nursed back to health."
     ],
     "fail_text": [
         "The kit is taken back to camp, but grows weak and dies a few days later.",
         "The patrol approaches the kit, only to realize that it's already gone. They swiftly bury it and move on."
     ],
-    "decline_text": "They decide to leave the kit alone.",
     "antagonize_text": "Your patrol moves the kit away from the territory, they have no interest in strange kits.",
     "antagonize_fail_text": "The kit smells of Twolegs, your patrol has no interest in what will surely grow to be a weak cat.",
     "chance_of_success": 40,
     "exp": 10,
     "win_skills": ["smart", "very smart", "extremely smart"],
-    "min_cats": 1,
+    "min_cats": 2,
     "max_cats": 6
 
+},
+{
+    "patrol_id": "gen_bord_newkitsolo3",
```

Comment:
in order to make sure this works for the patrol sprite, you should remove 'solo' from the ID and just increment the number